renpy/display/dragdrop.py

```diff
@@ -146,17 +146,26 @@ class Drag(renpy.display.displayable.Displayable, renpy.revertable.RevertableObj
         None.
 
     `clicked`
-        A callback this is called, with no arguments, when the Drag is
-        clicked without being moved. A droppable can also be focused
-        and clicked.  If the callback returns a value other than None,
-        that value is returned as the result of the interaction.
+        A callback that is called when the Drag is clicked without being moved.
+        It is called with one argument, the Drag being clicked on. A droppable
```

Comment:
To make it future-proof, maybe tell people that it should ignore unknown positional and keyword parameters ?